frontend/src/utils/helpers.js

```diff
@@ -169,3 +169,7 @@ export const formatComplianceCodeValueOrLabel = (code, showDescription) => {
 
   return `${section}${formattedSubSection}${formattedParagraph}${formattedSubParagraph}${formattedDescription}`;
 };
+
+export const getTableHeaders = (tableColumns) => {
+  return tableColumns.map((column) => column.title);
+};
```

Comment:
Totally fine with this implementation. If we'd like to leverage existing functions, lodash's `map` has this out-of-the-box: `_.map(tableColumns, 'title');`

https://lodash.com/docs/4.17.15#map

Reply:
good to know!